proc_IR.py

```diff
@@ -8,14 +8,9 @@
 #}}}
 coherence_pathway = {'ph1':0,'ph2':1}
 for thisfile,exp_type,nodename,postproc,f_range,t_range,clock_correction,IR,ILT in [
-       ('210610_water_TempCont_probe_DNP_1','ODNP_NMR_comp/ODNP',
-           'FIR_34dBm','spincore_IR_v1',
-           (0.1e3,0.345e3),(None,50e-3),True,False,False),
-        # ('210322_water_control_FIR_noPower','inv_rec','signal','spincore_IR_v1',
-        #    (-0.09e3,-0.06e3),(None,83e-3),False,False),
-        #('210517_4OHTempo_TempControl_probe_FIR_34dBm.','odnp_nmr_comp/inv_rec','signal','spincore_IR_v1',
-        #    (-0.4e3,0.4e3),(None,20e-3),False,False),
-        #('w3_201111','test_equip',2,'ag_IR2H',(-600,600),(0,None),True)
+       ('210615_S175R1a_pR_DDM_ODNP','odnp',
+           'FIR_0dBm','spincore_IR_v1',
+           (-0.5e3,0.5e3),(None,50e-3),True,False,False),
```

Comment:
this file should be an example, and this is pointing it to a different file, so it's not really a fix of the underlying code.